### DISCUSSION TITLE
Scaling fonts automatically in React Native WebView using Dynamic Type (iOS)

### DISCUSSION THREAD

**Discussion by ladp007** [react-native-webview/react-native-webview#2440] [Q&A]
I maintain a hybrid react-native (0.66.3) mobile app, which is basically a shell which wraps around our web apps very nicely. This method where we develop once and reuse on mobile and web platforms suits our use cases perfectly.

I noticed an issue in accessibility very recently. iOS has an accessibility feature to allow users to select their preferred text size which helps users who need larger text for better readability. (https://developer.apple.com/documentation/uikit/uifont/scaling_fonts_automatically)

Unfortunately, the iOS version of our hybrid react-native app doesn’t respect this setting and doesn't change its text size appropriately. The shell (native part) of the iOS app picks up the font size from dynamic type but the WebView (**react-native-webview: 7.5.1**) component in the iOS app doesn’t pick up the font size from dynamic type.

Is this a known issue/thing in “react-native-webview” or else could someone please kindly point me in the right direction (probably with an example) to add this accessibility feature to the iOS app?

**Note**: I posted this question to stackoverflow but didn't get any reply: https://stackoverflow.com/questions/71745135/change-fonts-size-automatically-in-react-native-webview-using-dynamic-type-ios

Thank you !!!

**Comment by oferRounds** [comment]
hi, seeing this issue also. Are there any plans to solve it?
